### ISSUE TITLE
Tooltip link broken in hominess

### ISSUE THREAD

**Issue by matkoniecz** [colouring-cities/colouring-core#1426]
![screen01](https://github.com/user-attachments/assets/01c2b6f7-bbae-4680-ae28-1ecd9e52870c)

**Comment by polly64** [comment]
@mdsimpson42 can you correct this on Tuesday am  confirm when completed here thanks

**Comment by matkoniecz** [comment]
@mdsimpson42 Have you had time to investigate what went wrong here?

**Comment by mdsimpson42** [comment]
The short answer is no. I missed the notification for this.

I can't replicate it. It appears to be working on the latest version of Colouring Britain both on production and in my local version.  

![image](https://github.com/user-attachments/assets/4d0b1d43-5bfc-4e7e-a2d1-f3ed0780735d)

If this is still happening, I will need more information. If its been resolved, then close the issue.

**Comment by matkoniecz** [comment]
![screen13](https://github.com/user-attachments/assets/14f2bf23-ceb0-4cf7-a635-21b3971b1e5c)

weird

I disabled adblock. I will try to disable all extensions and try with a Chrome. (I am using Firefox)

**Comment by mdsimpson42** [comment]
Confirmed. All the links in the tooltips in that section don't seem to work in Firefox, even with AdBlock disabled. I'll look into it. 

![image](https://github.com/user-attachments/assets/03463b29-0406-4008-9742-cc380791fc29)

**Comment by mdsimpson42** [comment]
Just checked and it works on localhost, but not on colouringbritain.org... 

![image](https://github.com/user-attachments/assets/aca907b7-245b-4b8f-8723-268047681d2e)
![image](https://github.com/user-attachments/assets/e4e0b69c-5b05-4233-986c-631c7338337f)

**Comment by mdsimpson42** [comment]
OK, so, I downloaded it and did some tests. I can confirm, I'm getting the bug in that version of the code.

- I thought it might be that the link was too long, so I tried a shorter URL, but it still happens.
- I thought it might be specific to the SliderDataEntry object, but I put a link in the tooltip of the 'How well do you know this building and its streetscape?' field, and that doesn't work either.
- I thought it might be category-specific, but it appears on other fields as well, such as UPRNs in the location category.

![image](https://github.com/user-attachments/assets/912e5899-8868-46f8-aded-3695d5f4cac5)

I tried running Firefox in Troubleshoot mode, with all mods disabled, but that didn't fix it. I've also been Googling, but I can't seem to find a solution.

-----

TLDR: None of the links in any of the tooltips are showing up correctly in Firefox, its not just limited to the paper links in the community section.

**Comment by matkoniecz** [comment]
has it worked in past for Firefox? if for some old code version it worked also in Firefox then bisecting may reveal when problem was introduced...
